base/sharedarray.jl

```diff
@@ -53,7 +53,7 @@ computation with the master process acting as a driver.
 If an `init` function of the type `initfn(S::SharedArray)` is specified, it is called on all
 the participating workers.
 """
-function SharedArray{T,N}(::Type{T}, dims::Dims{N}; init=false, pids=Int[])
+function SharedArray{T,N}(dims::Dims{N}; init=false, pids=Int[])
```

Comment:
Remove the definition on line 31 and move it here. This isn't a valid method definition at all. I.e. this does **NOT** define a method for `SharedArray{T,N}`.